src/sentry/stacktraces/functions.py

```diff
@@ -214,6 +215,10 @@ def process_generics(value, start):
             func_token = None
 
     if func_token:
+        if func_token.startswith("@") and platform in ("cocoa", "swift"):
+            # Found a Swift attribute instead of a function name, must be an
+            # anonymous function
+            func_token = "<anonymous function>"
```

Comment:
@jan-auer is `<anonymous function>` a good placeholder? Would something language-specific be better, e.g. "closure" (not even sure if every anon. function is called a closure)?

Reply:
Let's ask @philipphofmann how he would call this. Since you're detecting the platform, something language specific would probably be best.

Reply:
This is a complex topic. `thunk for @escaping @callee_guaranteed () -> ()` are keywords from the Swift Intermediate Language, see https://github.com/apple/swift/blob/main/docs/SIL.rst#function-types. To give you a better answer, I would need some time for investigation. Of course, you can as well read into that topic on your own.

Reply:
@brustolin, do you maybe have input on this one?